test/e2e/validation_test.go

```diff
@@ -143,3 +150,261 @@ func testNodeTaint(t *testing.T) {
 		assert.Equal(t, hasTaint, true, "expected Windows Taint to be present on the Node: %s", node.GetName())
 	}
 }
+
+// ensureTestRunnerSA ensures the proper ServiceAccount exists, a requirement for SSHing into a Windows node
```

Comment:
My thought process behind doing this was that it would make things easier when running the test locally, especially in situations where the resource specifications are being iterated on. Thinking about it more thats not the way that this should be done as it just complicates things, when like you implied, running in CI should be the main focus. Instead I have changed it so that if the resource already exists, these functions return nil. That maintains the ability to re-run these tests locally without too much headache, while removing the delete calls.